README.md

```diff
@@ -1,6 +1,26 @@
 # cluster-authentication-operator
+The authentication operator is an 
+[OpenShift ClusterOperator](https://github.com/openshift/enhancements/blob/master/enhancements/operator-dev-doc.md#what-is-an-openshift-clusteroperator).    
+It installs and maintains the Authentication [Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in a cluster and can be viewed with:     
+```
+oc get clusteroperator authentication -o yaml
+```
+
+The [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+`authentications.operator.openshift.io`    
+can be viewed in a cluster with:
+
+```console
+$ oc explain authentications.operator.openshift.io
```

Comment:
That's a human-readable description of the CR's fields, but the CRD can be viewed by `oc get crd authentications.operator.openshift.io -o yaml`

Reply:
ok i'll update that, and same, i am always confused by `authentication.{config,operator}`

Reply:
updated